stdlib/public/stubs/CommandLine.cpp

```diff
@@ -242,6 +242,39 @@ char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
 
   return argv;
 }
+#elif defined(__OpenBSD__)
+#include <sys/types.h>
+#include <sys/sysctl.h>
+#include <sys/exec.h>
+
+SWIFT_RUNTIME_STDLIB_API
+char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
+  assert(outArgLen != nullptr);
+  if (_swift_stdlib_ProcessOverrideUnsafeArgv) {
+    *outArgLen = _swift_stdlib_ProcessOverrideUnsafeArgc;
+    return _swift_stdlib_ProcessOverrideUnsafeArgv;
+  }
+
+  int mib[2] = {CTL_VM, VM_PSSTRINGS};
+  struct _ps_strings _ps;
+  size_t len = sizeof(_ps);
+
+  if (sysctl(mib, 2, &_ps, &len, NULL, 0) == -1) {
+    char **empty_argv = (char **)calloc(1, sizeof(char *));
+    empty_argv[0] = nullptr;
+    return empty_argv;
+  }
+
+  struct ps_strings *ps = (struct ps_strings *)_ps.val;
+  *outArgLen = ps->ps_nargvstr;
+
+  char **argv_copy = (char **)calloc(ps->ps_nargvstr + 1, sizeof(char *));
+  for(int i = 0; i < ps->ps_nargvstr; i++) {
+    argv_copy[i] = strdup(ps->ps_argvstr[i]);
+  }
+
+  return argv_copy;
```

Comment:
You need to store `nullptr` into the tail of `argv_copy` on your way out.

Reply:
I get that `calloc` will insert a zero bit pattern that will work here. It's still better to be explicit.